src/server.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
```

Comment:
The code below is for testing. You should be requiring the error-handlers, the logger and the validator files. Then when the server 'hears' '/person', it going to use the validator to check for a name. Look at the class-02 demo as an example.